lib/airbrake/capistrano/tasks.rb

```diff
@@ -6,7 +6,8 @@
   namespace :airbrake do
     desc "Notify Airbrake of the deploy"
     task :deploy do
-      on roles(:all) do
+      role = roles(:all, select: :primary).first || roles(:all).first
```

Comment:
I'm not sure why you need `.first` (it wasn't needed before). Could you please explain?